### ISSUE TITLE
[accessibility] The list marker links should be conveyed as buttons

### ISSUE THREAD

**Issue by Malvoz** [stefanocudini/leaflet-list-markers#14]
The `list-markers-li` list items contain `<a href="#">` elements, I suggest setting the [`role="button"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) attribute ([similar to Leaflet's zoom controls](https://github.com/Leaflet/Leaflet/blob/e94e69eb1aff885610365a894de0dfd2364bf505/src/control/Control.Zoom.js#L94), for example) such that they're conveyed as buttons and not regular links to screen reader users.

---

Other resources that may be helpful:

- [Leaflet guide to accessibility: Test your maps](https://leafletjs.com/examples/accessibility/#test-your-maps)
- [Leaflet Plugin Authoring Guide: Content Accessibility](https://github.com/Leaflet/Leaflet/blob/main/PLUGIN-GUIDE.md#content-accessibility)
